lib/addons/osb-addons.js

```diff
@@ -136,7 +136,7 @@ async function attach(pg_pool, app, addon_plan, service) {
   );
   try {
     await common.alamo.create_osb_bindings(
-      pg_pool, addon_plan.addon_service.id, addon_plan.id, addon_id, app.space, app.name,
+      pg_pool, addon_plan.addon_service.id, addon_plan.id, addon_id, app.space, app.name, app.id, service.service_attachment
```

Comment:
I think `app.id` should be removed

Reply:
otherwise it will use `app.id` for the `service_attachment` parameter and the `service.service_attachment` parameter will not be used